src/components/Menu.js

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from "react";
+import Hog from './Hog';
+
+const Menu = ({ hogs, greaseFilter, sortByWeight}) => {
```

Comment:
What would be a better approach would be to handle the filtering and sorting in the parent component and only pass in the filtered and sorted set of hogs. Less props and better separation of concerns.